### ISSUE TITLE
Foodsecurity harvester stuck updating the same dataset

### ISSUE THREAD

**Issue by georgiana-b** [NextGeoss/ckanext-nextgeossharvest#123]
This issue was pointed out by Jovanka but I was able to reproduce it as well.

When running on an empty database (as it would for example when we try to rebuild the indexes), if the `update_all` parameter is true, the harvester doesn't move on to the following days. It keeps updating the same dataset from the first day.

To reproduce: on `master`, run the foodsecurity harvester on a empty database with this configuration:

```
{
"start_date":"2017-01-01",
"collection":"FAPAR",
"username":"nextgeoss",
"password":"nextgeoss",
"make_private":false,
"update_all":true
}
```

Notice the behaviour from the second run further.

**Comment by davidcordeiro-deimos** [comment]
@georgiana-b This issue of the stuck harvester is solved in PR #130.

As a consequence of this approach, the harvester will also update the products collected in the previous job before collecting the new ones. Which will slow down the process in case of an empty database.

As a reminder, the purpose of the parameter `update_all` is to add forgotten fields to already collected products. So it is also suggested that if the database is empty, to set the parameter to false to avoid spending unnecessary processing power.

**Comment by JGulic** [comment]
Resolved with PR #130
